tests/query.json

```diff
@@ -200,7 +209,13 @@
         ["QueryCount", 3, "StringObject", "stringCol ==[c] 'a'"],
         ["QueryCount", 5, "StringObject", "stringCol BEGINSWITH[c] 'A'"],
         ["QueryCount", 4, "StringObject", "stringCol ENDSWITH[c] 'c'"],
-        ["QueryCount", 2, "StringObject", "stringCol CONTAINS[c] 'B'"]
+        ["QueryCount", 2, "StringObject", "stringCol CONTAINS[c] 'B'"],
+        ["QueryCount", 2, "StringObject", "stringCol LIKE[c] '*B?'"],
+        ["QueryCount", 0, "StringObject", "stringCol LIKE 1"],
+        ["QueryCount", 0, "StringObject", "stringCol LIKE LIKE"],
+        ["QueryCount", 0, "StringObject", "stringCol LIKE[A] '*BC'"],
```

Comment:
If it doesn't, I'll remove it and add it to the list of malformed queries in `parser.cpp`. It seems that if `"stringCol LIKE 1"` parses then something like this should parse too. I need to double-check.